src/chat.rs

```diff
@@ -1216,49 +1368,52 @@ pub fn get_chat_msgs(
 }
 
 /// Returns number of messages in a chat.
-pub fn get_msg_cnt(context: &Context, chat_id: u32) -> usize {
+pub fn get_msg_cnt(context: &Context, chat_id: ChatId) -> usize {
     context
         .sql
         .query_get_value::<_, i32>(
             context,
             "SELECT COUNT(*) FROM msgs WHERE chat_id=?;",
-            params![chat_id as i32],
+            params![chat_id],
         )
         .unwrap_or_default() as usize
 }
 
-pub fn get_fresh_msg_cnt(context: &Context, chat_id: u32) -> usize {
+pub fn get_fresh_msg_cnt(context: &Context, chat_id: ChatId) -> usize {
     context
         .sql
         .query_get_value::<_, i32>(
             context,
-            "SELECT COUNT(*) FROM msgs  \
-             WHERE state=10   \
-             AND hidden=0    \
-             AND chat_id=?;",
-            params![chat_id as i32],
+            "SELECT COUNT(*)
+               FROM msgs
+              WHERE state=10
+                AND hidden=0
+                AND chat_id=?;",
```

Comment:
Oh, I'm guessing the old stuff here was a rust-fmt converted block.  Which means rust-fmt does mess up the formatting style @r10s was using (and which i was trying to follow -probably badly- as he has much more sql experience) /cc @link2xt